addon/components/mobile/object-list-view.js

```diff
@@ -0,0 +1,63 @@
+/**
+  @module ember-flexberry
+*/
+
+import Ember from 'ember';
+import ObjectListView from './../objectlistview';
```

Comment:
Canyou rewrite it like "import ObjectListViewComponent from '../objectlistview';"?